README.md

```diff
@@ -1 +1,56 @@
-# ReactChamps
+# Stat Check
+
+[Lien du Trello](https://trello.com/b/uc8WwvZB/stat-check)
+
+## Goal
+This project is a way for me to practice and perfect my React programming skills. It is a learning tool that allows me to experiment and try various things out when it comes to front-end development. It also allows me to discover new technologies and learn to master them, all based around a theme that personnaly interests me : video games, and particularly League of Legends.
+
+## Tools used
+
+|  Software |  Version |  Link |
+|---|---|---|
+| Axios | 0.21.1  |  https://axios-http.com/docs/intro/ |
+| Material-UI | 4.11.2  |  https://material-ui.com/ |
+| MDI (Material Design Icons |5.9.55|https://materialdesignicons.com/|
+| React |17.0.1|https://fr.reactjs.org/|
+| React Router |5.2.0|https://reactrouter.com/web/guides/quick-start|
+| Typescript |4.1.5|https://www.typescriptlang.org/docs/|
+| Webpack |5.21.2| https://webpack.js.org/concepts/ |
+| Webpack Dev Server|3.11.2|https://webpack.js.org/configuration/dev-server/|
+
+## Choices
+
+I went for the use of **React Hooks** instead of class based components due to their general ease of execution concerning states (useState and useEffect in particular making it much simpler to program). Furthermore, being in a phase of learning React, I would rather start with a more efficient and powerful method of development without getting used to class-based component first, as it will almost force me to un-learn certain habits once I would switch to hooks.
+
+I choose to use **Material-UI** for its ease of setup and utilisation, which allows me to focus more on learning React and other techs rather than spend too much time on style etc...
```

Comment:
Actually, it is important for you to learn with libraries like material. And it also gives you what you can't learn through just react, and it is about choices of styles, visual concepts, colors and user experience. Keep in mind that you will not be just a "developer" but also, someone who will be able to choose colors, design, make mocks and be able to choose the best components to improve your user experience.